src/main/java/org/apache/commons/imaging/formats/tiff/photometricinterpreters/fp/PaletteEntryForRange.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.tiff.photometricinterpreters.fp;
```

Comment:
I am really bad with names. But I wonder if we could find an alternative for `fp`. See its parent package as example, `photometricinterpreters`. It could be `pi` too, but `tiff.pi.fp` looks even more enigmatic.

@gwlucastrig could we use some different name, easier for users/devs to know what's that package for/about?

Reply:
Originally, I was going to use something longer and more descriptive, but the length of the package line got a little too long.   But I agree that a more descriptive name is better. We could use photometricinterpreters.floatingPoint, but the capitalization disagrees with the conventions of the original package.  Do you prefer photometricinterpreters.floatingpoint?

Reply:
>Do you prefer photometricinterpreters.floatingpoint?

That'd be fine for me. While it's quite long, we have other packages with similar long names too, so at least that'd be consistent.